cmd/get_logs.go

```diff
@@ -203,6 +217,24 @@ func (glc *getLogsCmd) run() (err error) {
 		}
 	}
 	log.Infof("Logs downloaded to %s", glc.outputDirectory)
+	if glc.storageContainerSASURL != "" {
+		if glc.cs.Properties.IsCustomCloudProfile() && glc.cs.Properties.CustomCloudProfile.IdentitySystem == "adfs" {
+			log.Warn("Failed uploading logs for custom cloud with identity system of adfs")
```

Comment:
The term `Failed` is not properly communicating what's going on, the process is skipped.